src/test/java/uk/gov/companieshouse/api/accounts/service/impl/TnepValidationServiceImplTest.java

```diff
@@ -0,0 +1,120 @@
+package uk.gov.companieshouse.api.accounts.service.impl;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.client.MockRestServiceServer;
+import org.springframework.web.client.RestTemplate;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.springframework.test.web.client.match.MockRestRequestMatchers.method;
+import static org.springframework.test.web.client.match.MockRestRequestMatchers.requestTo;
+import static org.springframework.test.web.client.response.MockRestResponseCreators.withSuccess;
+
+public class TnepValidationServiceImplTest {
+
+    private TnepValidationServiceImpl tnepValidationService;
+    private MockRestServiceServer mockServer;
+
+    private static final String S3_TESTLOCATION = "s3://testlocation/testdoc";
+    private static final String TNEP_URL = "http://testtnep.companieshouse.gov.uk/validate";
+
+    private static final String VALIDATION_FAILURE_RESPONSE =
```

Comment:
probably worth putting this xml in its own xml file in the test resources folder